prometheus_all/input.tf

```diff
@@ -124,6 +129,11 @@ variable "alertable_postgres_services" {
   default     = {}
 }
 
+variable "alertable_apps" {
+  description = "List of apps that will have alerting enabled. [\"space1/app1\", \"space2/app2\", ...]"
```

Comment:
The name of the app is unique in the whole paas so the space is not required here